chainerx_cc/chainerx/python/routines.cc

```diff
@@ -9,6 +9,7 @@
 
 #include <nonstd/optional.hpp>
 
+#include "chainerx/routines/activation.h"
```

Comment:
The includes should be ordered lexicographically. Could you fix it? Usually, it's fixed automatically if you apply clang-format 🤔